triagebot.toml

```diff
@@ -91,6 +91,29 @@ exclude_labels = [
     "requires-nightly",
 ]
 
+[autolabel."T-rustdoc"]
+trigger_files = [
+    # Source code
+    "src/librustdoc",
+    "src/tools/rustdoc",
+    "src/rustdoc-json-types",
+
+    # Tests
+    "src/test/rustdoc",
+    "src/test/rustdoc-ui",
+    "src/test/rustdoc-gui",
+    "src/test/rustdoc-js",
+    "src/test/rustdoc-js-std",
+    "src/test/rustdoc-json",
```

Comment:
In practice we're currently using a str::starts_with, so just src/test/rustdoc is technically enough, but this seems harmless so likely OK (and we may want to alter that to a Path prefix).

Reply:
Ah, didn't realize that! It's probably good to have the paths not rely on that behavior though :)